administrate/zone-migration.md

```diff
@@ -0,0 +1,120 @@
+
+For most of the services, you will be able to migrate them as-is. For others, you will need to create them and do a manual migration.
+
+Keep in mind that our support team remains available, should you have any questions regarding zone migration.
+
+## Application migration
+
+Most of the applications will migrate without issues. If one of your applications matches any criteria is in the list below, you will need extra steps
+described in the following sub-sections.
+
+- Any application that uses a FSBucket add-on
+- PHP applications
+
+In this case, please read the prerequisites below first and then follow the right sub-section.
+
+### Prerequisites
+
+Besides redeploying your application in your target zone, you have to keep a few things in mind:
+- Increased latency might be observed during DNS propagation (more below)
+- You will need to update your DNS settings, so make sure you have access to your DNS registrar
+
+One of the first things you can do is to lower your DNS records TTL (time to live). in order to speed up the DNS propagation during the update of your domains DNS records.
+
+On your registrar's interface, find your application's domain. Usually, the DNS record will be `domain.tld IN CNAME domain.<current-zone>.clever-cloud.com` with a certain TTL.
+Edit that TTL to 60 seconds instead of the existing value, and wait for this change to propagate (wait as much time as the old TTL value). This change will allow a faster DNS propagation when
+you will have to update the `CNAME` record value to `domain.<target-zone>.clever-cloud.com`, lowering the amount of time your users will experience an increased latency.
+
+{{< alert "info" "Note" >}}
+After the migration, you can set back your old TTL value.
+{{< /alert >}}
+
+#### Increased latency
+
+Once your application has been redeployed to your target zone, your application's domain will still have its DNS pointing to the old zone. As a consequence, the
+latency may increase for your users. For example, if you are migrating from Paris (PAR) to Montreal (MTL), your visitors will connect to our Paris infrastructure, which will
+then redirect the trafic to our Montreal infrastructure, leading to requests taking more time.
+
+You might also experience an increased latency between your application and other services (like a PostgreSQL add-on for example). If your application has switched to the Montreal zone
+and your PostgreSQL add-on is still in Paris, your application will need extra time to query the database.
+
+#### DNS updates
+
+Once your application has been migrated to your target zone, you will find in the `Domain names` section the new `A` and `CNAME` DNS records to use for your application's domains.
+If you were using `A` records, update the records (there might be more or fewer of them than what you had previously, this is normal). If you used a `CNAME` record, simply change the value.
```

Comment:
```suggestion
If you were using `A` records, update the records (there might be more or fewer of them than what you had previously, this is normal). If you used a `CNAME` record, simply changes the value.
```

Reply:
This doesn't feel right, I'm omitting it!